src/signatures.rs

```diff
@@ -1371,6 +1374,9 @@ signatures! {
     format = BdavMpeg2TransportStream
     value = b"\x47" offset = 4, b"\x47" offset = 196
 
+    format = Commodore64Program
+    value = b"\x01\x08"
```

Comment:
It seems that byte at offset 6 is always \x9E (this is also checked by the file command)

Reply:
Updated and moved signature to 3-byte section